EventsExpress/ClientApp/src/components/location/location.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
```

Comment:
I would recommend moving this over to the field-renderers package and group with all required utils. 
smth like:
```
form-helpers/
- location/
-- location-map-with-marker.jsx
-- event-location-renderer.jsx
-- index.jsx
```

Reply:
resolved